axiom/nr_log_events_private.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file contains log events data internals.
+ */
+#ifndef NR_LOG_EVENTS_PRIVATE_HDR
+#define NR_LOG_EVENTS_PRIVATE_HDR
+
+//#include "nr_log_event.h"
```

Comment:
```suggestion
```

Reply:
Missed something else - thanks for taking the time to catch these.